src/Context/AppContext.tsx

```diff
@@ -15,12 +16,20 @@ type Props = {
 };
 
 export const AppContextProvider = ({ children }: Props) => {
-  const { employees, isLoading } = useFetchEmployees();
+  const { employees: fetchedEmployees, isLoading } = useFetchEmployees();
+  const [employees, setEmployees] = useState<Employee[]>([]);
+
+  useEffect(() => {
+    if (fetchedEmployees.length > 0) {
+      setEmployees(fetchedEmployees);
+    }
+  }, [fetchedEmployees]);
```

Comment:
there you should check fetchedEmployees.length because array ref could be the same as earlier

Reply:
I think it's ready but I'm not sure...

Reply:
still you have in array dependency fetchedEmployeed but you should have fetchedEmployees.length

Reply:
tell me that's what it was about...